### ISSUE TITLE
Errors since updating add-on

### ISSUE THREAD

**Issue by Danika-Dakika** [Vocab-Apps/anki-hyper-tts#202]
I updated the add-on about 3 days ago (to 1.12.5), but I haven't used it since then. Today when I tried to run a specific Preset, it worked the first time, but the 2nd and subsequent times, I got this error (different voice_id each time, because I have it set to random across about a dozen voices) -- 

> Encountered an error while Adding Audio to Note: Voice not found: TtsVoiceId_v3(voice_key={'model_id': 'eleven_multilingual_v2', 'voice_id': 'pMsXgVXv3BLzUgSXRplE'}, service='ElevenLabs')]

If I close and re-open the "Preset Rules" window, I can sometimes get it to generate another audio file, sometimes not. 

----

When I click `Edit` from there to check the configuration, I get another generic error [Copy Debug Info added below] -- 

![image](https://github.com/user-attachments/assets/8cfc0f34-2339-4e84-857f-58070c50ed44)
----

When I try to open any of my Collection Audio presets from the HyperTTS menu, I get errors like this -- 
![image](https://github.com/user-attachments/assets/106ea4c3-581c-4c19-ae04-41f60e76aa1d)

----

```
Anki 24.06.2 (33a92379)  (ao)
Python 3.9.18 Qt 6.6.2 PyQt 6.6.1
Platform: Windows-10-10.0.22621

Traceback (most recent call last):
  File "C:\Users\danik\AppData\Roaming\Anki2\addons21\111623432\component_mappingrule.py", line 137, in edit_button_clicked
    component_batch.create_dialog_editor_existing_preset(self.hypertts, self.editor_context, self.model.preset_id)
  File "C:\Users\danik\AppData\Roaming\Anki2\addons21\111623432\component_batch.py", line 546, in create_dialog_editor_existing_preset
    dialog.configure_editor_existing_preset(editor_context, preset_id)
  File "C:\Users\danik\AppData\Roaming\Anki2\addons21\111623432\component_batch.py", line 514, in configure_editor_existing_preset
    batch_component.load_preset(preset_id)
  File "C:\Users\danik\AppData\Roaming\Anki2\addons21\111623432\component_batch.py", line 107, in load_preset
    self.load_model(model)
  File "C:\Users\danik\AppData\Roaming\Anki2\addons21\111623432\component_batch.py", line 117, in load_model
    self.voice_selection.load_model(model.voice_selection)
  File "C:\Users\danik\AppData\Roaming\Anki2\addons21\111623432\component_voiceselection.py", line 111, in load_model
    self.redraw_selected_voices()
  File "C:\Users\danik\AppData\Roaming\Anki2\addons21\111623432\component_voiceselection.py", line 484, in redraw_selected_voices
    voice = self.hypertts.service_manager.locate_voice(voice_id)
  File "C:\Users\danik\AppData\Roaming\Anki2\addons21\111623432\servicemanager.py", line 226, in locate_voice
    raise errors.VoiceIdNotFound(voice_id)
111623432.errors.VoiceIdNotFound: Voice not found: TtsVoiceId_v3(voice_key={'model_id': 'eleven_multilingual_v2', 'voice_id': 'ThT5KcBeYPX3keUQqHPh'}, service='ElevenLabs')]

===Add-ons (active)===
(add-on provided name [Add-on folder, installed at, version, is config changed])
Anki Simulator ['817108664', 2023-11-06T09:26, 'None', '']
AnkiWebView Inspector ['31746032', 2023-06-27T12:26, 'None', '']
Basic Printing Support ['1025789669', 2023-10-18T19:57, 'None', '']
FSRS Helper Postpone  Advance  Load Balance  Easy Days  Disperse Siblings ['759844606', 2024-07-09T05:54, 'None', mod]
HyperTTS - Add speech to your flashcards ['111623432', 2024-08-11T23:48, 'None', mod]
Image Occlusion Enhanced ['1374772155', 2022-04-09T00:15, 'None', '']
Inline media ['683715045', 2022-07-24T16:24, 'None', mod]
Remove card history ['2089200096', 2023-10-18T20:15, 'None', '']
Review Heatmap ['1771074083', 2022-06-29T18:43, 'None', '']

===IDs of active AnkiWeb add-ons===
1025789669 111623432 1374772155 1771074083 2089200096 31746032 683715045 759844606 817108664

```

**Comment by luc-vocab** [comment]
Are you able to remove the voice in question from your config and restart ? Let me know if the question is not clear and I can clarify or even help you do it.

**Comment by luc-vocab** [comment]
Hi, I've just released an update to HyperTTS (1.12.7) which handles this better and omits voices which are not found when in random or priority mode.

**Comment by Danika-Dakika** [comment]
Thanks for this fix! I just got around to updating everything and it's working great. I went through the ElevenLabs voices and replaced all the old ones with new ones for all of my presets.
